geode-dunit/src/main/java/org/apache/geode/cache30/MultiVMRegionTestCase.java

```diff
@@ -8745,46 +8284,63 @@ public void run() {
       fail("neither member saw event conflation - check stats for " + name);
     }
 
-    // check consistency of the regions
-    Map r1Contents = (Map) vm1.invoke(() -> this.getCCRegionContents());
-    Map r2Contents = (Map) vm2.invoke(() -> this.getCCRegionContents());
-    Map r3Contents = (Map) vm3.invoke(() -> this.getCCRegionContents());
+    // Wait for the members to eventually be consistent
+    // For no-ack regions, messages may still be in flight between replicas at this point
+    waitAtMost(5, MINUTES).until(() -> {
 
-    for (int i = 0; i < 10; i++) {
-      String key = "cckey" + i;
-      assertEquals("region contents are not consistent for " + key, r1Contents.get(key),
-          r2Contents.get(key));
-      assertEquals("region contents are not consistent for " + key, r2Contents.get(key),
-          r3Contents.get(key));
-      for (int subi = 1; subi < 3; subi++) {
-        String subkey = key + "-" + subi;
-        if (r1Contents.containsKey(subkey)) {
-          assertEquals("region contents are not consistent for " + subkey, r1Contents.get(subkey),
-              r2Contents.get(subkey));
-          assertEquals("region contents are not consistent for " + subkey, r2Contents.get(subkey),
-              r3Contents.get(subkey));
-        } else {
-          assertTrue(!r2Contents.containsKey(subkey));
-          assertTrue(!r3Contents.containsKey(subkey));
+      // check consistency of the regions
+      Map r1Contents = vm1.invoke(MultiVMRegionTestCase::getCCRegionContents);
+      Map r2Contents = vm2.invoke(MultiVMRegionTestCase::getCCRegionContents);
+      Map r3Contents = vm3.invoke(MultiVMRegionTestCase::getCCRegionContents);
+
+      try {
+        for (int i = 0; i < 10; i++) {
```

Comment:
The 10 is in the original code. Our focus was to reduce the warnings. To maintain that focus, we deliberately (and often with difficulty) refrained from otherwise improving the code.

Reply:
GEODE-5669 has been submitted to address the usage of/removal of fail

Reply:
GEODE-5669 address the fail situation.